scripts/travis-functest.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
```

Comment:
Add -e to the end?

In the older case when each of the following script lines fail, travis would exit with an error, now as these are moved to a script, travis would check final exit state of the script as I understand. Hence to catch failures of each command/script invocation and fail immediately we should add the -x. I have not tested this though.

Reply:
done